build/ci.go

```diff
@@ -268,14 +273,11 @@ func doInstall(cmdline []string) {
 
 func buildFlags(env build.Environment) (flags []string) {
 	var ld []string
+	ld = append(ld, "-s", "-w")
```

Comment:
I don't think this is a good change. I'm not even sure why it's disabled on darwin? (cc @fjl) We need the debug symbols to interpret crash logs, otherwise we'll get useless bugreports.

Reply:
Apparently Go is a bit smarter and still keeps a symbol stub for crash dumps.

Reply:
The symbol table is mostly for external debuggers. Without it external debugging with tools like 'delve' is not possible.  I can agree with adding `-trimpath` because it helps with reproducible builds.

Reply:
@fjl Any particular reason that -s was only added for macos, not the rest? I've just tried it on Linux and it does cut off 10MB of junk + stack traces and pprof still work.

Reply:
It was added for macOS in this commit: https://github.com/ethereum/go-ethereum/commit/1cf2ee4597e3e1902088fd84663d4ce4eff8946a

Reply:
I know -s makes the binary smaller, but as I said, stripping the debug information has downsides for debugging.